backend/src/main/java/org/cryptomator/hub/api/ValidationPatterns.java

```diff
@@ -1,11 +1,10 @@
 package org.cryptomator.hub.api;
 
-public final class ValidationUtil {
+public final class ValidationPatterns {
 
 	public static final String ID_PATTERN = "[-\\w]+";
-	public static final String JWX_PATTERN = "[-=.\\w]+";
 	public static final String NAME_PATTERN = "(?U)[-_\\p{Alpha}\\p{Digit}\\x20]+";
```

Comment:
is this still used?

Reply:
@infeo is it?

Reply:
ow, sh*te, completely forgot. Will fix it.

Reply:
wait until you see my other comment regarding the removal of `@ValidName`, though 😜

Reply:
Done in [002a11c](https://github.com/cryptomator/hub/pull/154/commits/002a11ccf5b2cdcc5b6be018d76304bfe167ab25)